tests/integration/hosts/maya/test_publish_in_maya.py

```diff
@@ -0,0 +1,100 @@
+import pytest
+import os
+import shutil
+
+from tests.lib.testing_classes import PublishTest
+
+
+class TestPublishInMaya(PublishTest):
+    """Basic test case for publishing in Maya
+
+        Uses generic TestCase to prepare fixtures for test data, testing DBs,
+        env vars.
+
+        Opens Maya, run publish on prepared workile.
+
+        Then checks content of DB (if subset, version, representations were
+        created.
+        Checks tmp folder if all expected files were published.
+
+    """
+    PERSIST = True
+
+    TEST_FILES = [
+        ("1pOwjA_VVBc6ooTZyFxtAwLS2KZHaBlkY", "test_maya_publish.zip", "")
+    ]
+
+    APP = "maya"
+    APP_VARIANT = "2019"
```

Comment:
Apparently 2019 is using no one from our clients. Recommended version is 2020 but some of clients is using 2018. We have to develop a way to test it on list of versions.

Reply:
Agree, but I would put it in next PR, where I need to create some more intelligent runner of test classes.

Reply:
> Apparently 2019 is using no one from our clients. Recommended version is 2020 but some of clients is using 2018. We have to develop a way to test it on list of versions.

Just to note. We use 2019.